README.md

```diff
@@ -20,6 +20,9 @@
   <a href="https://github.com/fetchai/cosmpy/actions/workflows/workflow.yml">
     <img alt="AEA framework sanity checks and tests" src="https://github.com/fetchai/cosmpy/actions/workflows/workflow.yml/badge.svg">
   </a>
+  <a href="https://github.com/fetchai/cosmpy/actions/workflows/workflow.yml">
```

Comment:
I am not sure should we use it or not, but for review purposes, I added it. 
It's optional. it won't add any impact.